### PR TITLE
Merge dev into release 

### PR DESCRIPTION
**Pull request by jessgilbert**
HPO section only appears when available in active language and added Investor Support directory section
image

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 ## Pre-release
+- [CI-103](https://uktrade.atlassian.net/browse/CI-103) Whole HPO section disappears if not available in active language
```
Comment (MichaelWalker) [inline CHANGELOG.md]: If we are merging into release, do we need to change `Pre-release` to the actual release tag?

File: core/views.py [hunk 2]
```diff
@@ -66,10 +67,14 @@ class LandingPageCMSView(GetCMSComponentMixin, CMSPageView):
     subpage_groups = ['sectors', 'guides']
 
     def get_context_data(self, **kwargs):
+        pages = self.page['high_potential_opportunities'],
         return super().get_context_data(
             international_home_page_link=(
                 urls.GREAT_INTERNATIONAL
             ),
+            show_hpo_section=bool(
+                pages and filter_by_active_language(pages[0])
```
Comment (MichaelWalker) [inline core/views.py]: Nice - I like this solution better than using the filters!
Would it be worth adding a test to check that this property works?

File: CHANGELOG.md (hunk 1 above)
Comment (jessgilbert) [reply CHANGELOG.md]: Do I do that? How would I do that without putting it into dev first?

File: core/templates/core/landing_page.html [hunk 3]
```diff
@@ -160,7 +187,7 @@ <h2 class="heading-large">{{ page.how_we_help_title }}</h2>
 				<h2 class="heading-large">{{ page.contact_section_title }}</h2>
 				<p class="font-small margin-bottom-45">{{ page.contact_section_content }}</p>
 				<div>
-					<a href={{ page.contact_section_call_to_action_url }} class="button button-arrow-small">{{ page.contact_section_call_to_action_text }}</a>
+					<a href="{{ page.contact_section_call_to_action_url }}" class="button button-arrow-small">{{ page.contact_section_call_to_action_text }}</a>
 				</div>
 			</div>
 		</div>
```
Comment (Kenneth-Siu) [inline core/templates/core/landing_page.html]: You guys have a JIRA board or anything to log tasks?

This file has a mix of tabs and spaces — something to remember to do but not something that needs to be done for this.

Comment (jessgilbert) [reply core/templates/core/landing_page.html]: Yeah we have a Jira board https://uktrade.atlassian.net/secure/RapidBoard.jspa?rapidView=142&selectedIssue=CI-108 
I have referenced the jira tickets in the branches.
Ah sorry about that, I can fix it now if you want? My pycharm doesn't work that well and so it doesn't really format html very well

**Pull request merged**